interpreter/k8s/src/main/scala/io/buoyant/interpreter/k8s/IstioInterpreter.scala

```diff
@@ -2,14 +2,17 @@ package io.buoyant.interpreter.k8s
 
 import com.twitter.finagle._
 import com.twitter.finagle.naming.NameInterpreter
+import io.buoyant.k8s.SingleNsNamer
 import io.buoyant.k8s.istio.RouteCache
 import io.buoyant.namer.ConfiguredDtabNamer
 import istio.proxy.v1.config.RouteRule
 
 object IstioInterpreter {
   private val istioPfx = "/#/io.l5d.k8s.istio"
+  private val k8sPfx = "/#/io.l5d.k8s.ns"
   private val defaultRouteDtab = Dtab.read(s"""
     |/svc/ext => /$$/inet ;
```

Comment:
I think our updated logic should be:

* If matching route exists: /svc/route (send to target service)
* If cluster exists but no matching route: /svc/dest (send to target service; fallback to istio-egress)
* If cluster does not exist: /svc/ext (send to istio-egress)